PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringAppViewModel.java

```diff
@@ -0,0 +1,263 @@
+        this.clusterName = clusterName;
+    }
+
+    public String getAppName() {
+        return appName;
+    }
+
+    public void setAppName(String appName) {
+        this.appName = appName;
+    }
+
+    public String getRuntimeVersion() {
+        return runtimeVersion;
+    }
+
+    public void setRuntimeVersion(String runtimeVersion) {
+        this.runtimeVersion = runtimeVersion;
+    }
+
+    public Integer getCpu() {
+        return cpu;
+    }
+
+    public void setCpu(Integer cpu) {
+        this.cpu = cpu;
+    }
+
+    public Integer getMemoryInGB() {
+        return memoryInGB;
+    }
+
+    public void setMemoryInGB(Integer memoryInGB) {
+        this.memoryInGB = memoryInGB;
+    }
+
+    public String getJvmOptions() {
+        return jvmOptions;
+    }
+
+    public void setJvmOptions(String jvmOptions) {
+        this.jvmOptions = jvmOptions;
+    }
+
+    public boolean isEnablePersistentStorage() {
+        return enablePersistentStorage;
+    }
+
+    public void setEnablePersistentStorage(boolean enablePersistentStorage) {
+        this.enablePersistentStorage = enablePersistentStorage;
+    }
+
+    public boolean isEnablePublicUrl() {
+        return enablePublicUrl;
+    }
+
+    public void setEnablePublicUrl(boolean enablePublicUrl) {
+        this.enablePublicUrl = enablePublicUrl;
+    }
+
+    public Map<String, String> getEnvironment() {
+        return environment;
+    }
+
+    public void setEnvironment(Map<String, String> environment) {
+        this.environment = environment;
+    }
+
+    public String getPublicUrl() {
+        return publicUrl;
+    }
+
+    public void setPublicUrl(String publicUrl) {
+        this.publicUrl = publicUrl;
+    }
+
+    public String getTestUrl() {
+        return testUrl;
+    }
+
+    public void setTestUrl(String testUrl) {
+        this.testUrl = testUrl;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public void setStatus(String status) {
+        this.status = status;
+    }
+
+    public Integer getUpInstanceCount() {
+        return upInstanceCount;
+    }
+
+    public void setUpInstanceCount(Integer upInstanceCount) {
+        this.upInstanceCount = upInstanceCount;
+    }
+
+    public Integer getDownInstanceCount() {
+        return downInstanceCount;
+    }
+
+    public void setDownInstanceCount(Integer downInstanceCount) {
+        this.downInstanceCount = downInstanceCount;
+    }
+
+    public Integer getTotalStorageInGB() {
+        return totalStorageInGB;
+    }
+
+    public void setTotalStorageInGB(Integer totalStorageInGB) {
+        this.totalStorageInGB = totalStorageInGB;
+    }
+
+    public Integer getUsedStorageInGB() {
+        return usedStorageInGB;
+    }
+
+    public void setUsedStorageInGB(Integer usedStorageInGB) {
+        this.usedStorageInGB = usedStorageInGB;
+    }
+
+    public boolean isCanStart() {
+        return canStart;
+    }
+
+    public void setCanStart(boolean canStart) {
```

Comment:
do we need to add `canStart` `canStop` methods in Model? As these valued could get from status

Reply:
will consider to refactor it later